libs/wingsdk/src/target-tf-azure/bucket.ts

```diff
@@ -52,6 +54,7 @@ export class Bucket extends cloud.Bucket {
   public readonly storageContainer: StorageContainer;
   private readonly public: boolean;
   private readonly storageAccount: StorageAccount;
+  protected _clientClass = BucketClient;
```

Comment:
Why is this needed?

Reply:
When going over the bound host operations, we check if this client class implements all of the used operations. This keeps us from saving a dedicated list of non-implemented methods (that will have to be updated manually for each change- had to maintain). This approach introduces 0 maintenance- in the price of having the client class implement `Partial<ClientInterface>` and having the client class specified here. (I added tests that can show easily how it's done)

Reply:
if you have another idea, please let me know :)

Reply:
another option I can think of is to implement a different list of `_getInflightOps` in each of the targets (keeping the parent's one abstract maybe), which still have some maintenance